### PR TITLE
Haskell-CI: use preinstalled packages

### PR DESCRIPTION
**Pull request by andreasabel**

### CODE REVIEW

**Review by Mikolaj — APPROVED** [review]
+1

File: cabal.haskell-ci [hunk 1]
```diff
@@ -1,4 +1,5 @@
 branches: master
+installed: +all
```
Comment (andreasabel) [inline cabal.haskell-ci:2]: This seems to anyway be the default (I hadn't been sure about this), so this line changes nothing.

File: cabal.haskell-ci [hunk 2]
```diff
@@ -19,7 +20,7 @@ constraint-set no-lukko
   constraints: hackage-security -lukko
 
 constraint-set containers-0.7
-  ghc: >= 8.2 && < 9.8
+  ghc: >= 8.2 && < 9.10
```
Comment (andreasabel) [inline cabal.haskell-ci:23]: This change is worth keeping.

File: cabal.haskell-ci (hunk 1 above)
Comment (Mikolaj) [reply cabal.haskell-ci:2]: Good to know.

**Pull request merged**